sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterLoggingExtensions.cs

```diff
@@ -31,10 +31,6 @@ public static OpenTelemetryLoggerOptions AddAzureMonitorLogExporter(this OpenTel
                 throw new ArgumentNullException(nameof(loggerOptions));
             }
 
-            // Ideally user should set this to true
```

Comment:
There are many tests that uses `ParseStateValues`. Should we remove those tests?

Reply:
Let me check and see what the impact is

Reply:
This is safe to remove from most of our tests. We had two tests that were asserting true & false. I combined that into one test using `Theory`